Task50/UnitTest1.cs

```diff
@@ -0,0 +1,209 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Threading;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+
+namespace Task50
+{
+    public class Tests
+    {
+        IWebDriver _driver;
+
+        //explicit waiter for the webelement
+        bool wait(IWebDriver driver, By locator)
+        {
+            WebDriverWait waiter = new WebDriverWait(_driver, TimeSpan.FromSeconds(10));
+            waiter.PollingInterval = TimeSpan.FromMilliseconds(100); //changing polling frequency
+
+            try
+            {
+                var element = waiter.Until(condition =>
+                {
+                    try
+                    {
+                        var elementToBeDisplayed = _driver.FindElement(locator);
+                        return elementToBeDisplayed.Displayed;
+                    }
+                    catch (NoSuchElementException)
+                    {
+                        return false;
+                    }
+                    catch (StaleElementReferenceException)
+                    {
+                        return false;
+                    }
+
+                });
+
+                return true;
+            }
+            catch (WebDriverTimeoutException)
+            {
+                return false;
+            }
+        }
+
+        // making screenshot, saving to the file in jpeg format
+        void MakeScreenShot(IWebDriver driver, string path)
+        {
+            Screenshot img = ((ITakesScreenshot)driver).GetScreenshot();
+            img.SaveAsFile(path, ScreenshotImageFormat.Png);
+        }
+
+        [SetUp]
+        public void Setup()
+        {
+            _driver = new ChromeDriver();
+        }
+
+        [TearDown]
+        public void CleanUp()
+        {
+            _driver.Close();
+        }
+
+        [TestCase("seleniumtests@tut.by", "123456789zxcvbn")]
+        [TestCase("seleniumtests2@tut.by", "123456789zxcvbn")]
+
+        [Test]
+        public void LoginTest(string loginstr, string passwordstr)
+        {
+            _driver.Navigate().GoToUrl("https://mail.tut.by");
+
+            MakeScreenShot(_driver, "/Users/macbookpro/Desktop/TestScreenshot.png"); //making screenshot, saving it to the Desktop
```

Comment:
It's not recommended to use dependency path in code. Your MakeScreenshot method will fail in case I run your tests on my local machine. Just put images and other any output files from your code in solution root. Use Environment.CurrentDirectory or whatever you want (there are several ways how to get project base location)